src/screenComponents/scanningDialog.h

```diff
@@ -12,7 +13,7 @@ class GuiButton;
 class GuiScanningDialog : public GuiElement
 {
 private:
-    static constexpr int max_sliders = 4;
+    static constexpr int max_sliders = EScanningComplexity::SC_MAX;
```

Comment:
SC_MAX isn't the same thing as max_sliders. They are the same value by luck, not by design.

Reply:
addressed